docs/xmlprofiles.rst

```diff
@@ -690,9 +690,8 @@ This section of the :class:`Participant's rtps` configuration allows defining bu
 +---------------------------+---------------------------------------+-------------------------+-----------------------+
 | Name                      | Description                           | Values                  | Default               |
 +===========================+=======================================+=========================+=======================+
-| ``<use_SIMPLE_RTPS_PDP>`` | Indicates if the Participant          | ``Boolean``             | :class:`true`         |
-|                           | must use the Simple RTPS Discovery    |                         |                       |
-|                           | Protocol.                             |                         |                       |
+| ``<discoveryProtocol>``   | Indicates which kind of PDP protocol  | ``Boolean``             | :class:`true`         |
```

Comment:
Lacks `<discovery_config>` node. Some tag has been moved to his new node. `<discoveryProtocol>` isn't boolean, it is an enum.
Document it properly.